addons/web/static/tests/fields/basic_fields_tests.js

```diff
@@ -4806,6 +4815,227 @@ QUnit.module('basic_fields', {
         assert.strictEqual(form.$('.o_progressbar_value').text(), '999 / 5',
             'The value of the progress bar should be correct after the update');
 
+        $('.o_form_button_save').click();
+
+        form.destroy();
+    });
+
+    QUnit.test('Field ProgressBar: value should update in readonly mode', function (assert) {
```

Comment:
not sure about this feature

Reply:
@aab-odoo That's the whole point :D
I'm not sure either how this widget should behave, I made some guesses along the way

Reply:
We should ask NGR, but in a form view, it shouldn't do that imho